integrationTests/mock/oneSCExecutorMockVM.go

```diff
@@ -218,6 +222,55 @@ func (vm *OneSCExecutorMockVM) processAddFunc(input *vmcommon.ContractCallInput,
 	}, nil
 }
 
+func (vm *OneSCExecutorMockVM) processWithdrawFunc(input *vmcommon.ContractCallInput, value *big.Int) (*vmcommon.VMOutput, error) {
```

Comment:
The "withdraw" is seen from the receiverAccount or the smartContractAccount? I want to send to the smart contract some money, or withdraw some money from the SC?